client/src/pages/SearchResultsPage.jsx

```diff
@@ -4,6 +4,7 @@ import { Stack, Typography } from "@mui/material";
 import BasePage from "./BasePage";
 import FiltersBar from "../components/FiltersBar";
 import SearchBar from "../components/SearchBar";
+
```

Comment:
Extra line, is this necessary?

Reply:
We'll ignore this for now, but typically, we do not need/want to commit extra unnecessary changes

Reply:
Sorry about that, I thought I was sure to have removed it. Done